src/test/java/com/egov/core-services/pretests/updateUserPretest.feature

```diff
@@ -0,0 +1,187 @@
+Feature: Update user profile
+
+
+Background:
+     * def jsUtils = read('classpath:jsUtils.js')
+     # Calling access token
+     * def authUsername = superUserUserName
+     * def authPassword = superUserPassword
+     * def tenantId = 'pb.jalandhar'
+     * print tenantId
+     * def authUserType = 'EMPLOYEE'
```

Comment:
please store tenantId and authUserType in yaml constants file

Reply:
@yatheendra-egov I have addressed this comment and updated the PR as well.